packages/monitors/test/OptimisticOracleContractMonitor.js

```diff
@@ -243,6 +244,20 @@ describe("OptimisticOracleContractMonitor.js", function () {
     assert.isTrue(lastSpyLogIncludes(spy, `https://etherscan.io/address/${optimisticRequester.options.address}`));
     assert.isTrue(lastSpyLogIncludes(spy, `https://etherscan.io/tx/${requestTxn.transactionHash}`));
 
+    assert.isTrue(
+      lastSpyLogIncludes(
```

Comment:
this isnt checking the `|${linktext}` part ( it would still pass with or without it), not sure if that matters but mentioning it in case this was an oversight

Reply:
True, I was more worried about the link being in the logs than testing exactly what the link says.

I think the exact text in the log isn't something we should test, only that the log contains the information we care about.

But if you disagree, I'm happy to add tests for this.

Reply:
agreed. this is how we do it in other places. you just care about the key info, not the exact string. exact strings make it painful if we choose the change the layout.

Reply:
Yea sounds good